package/python-kiwi-spec-template

```diff
@@ -187,7 +187,9 @@ Recommends:     gfxboot
 Requires:       grub2-efi-x64
 %endif
 %endif
+%if 0%{?fedora} || 0%{?rhel} || 0%{?suse_version}
```

Comment:
```suggestion
%if ! (0%{?debian} || 0%{?ubuntu})
```

Reply:
yes that's a better read, thanks I adapted to it